.gitignore

```diff
@@ -22,3 +22,4 @@ monicadump.sql
 .sentry-release
 .phpunit.result.cache
 .composer
+.idea/
```

Comment:
Do not add a specific host application here, use a global [gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) instead.

Reply:
I removed this line